src/planscape/plan/models.py

```diff
@@ -102,3 +103,20 @@ class ProjectArea(models.Model):
     # The sum total of the project area geometries.
     estimated_area_treated: models.IntegerField = models.IntegerField(
         null=True)
+
+
+class ConditionScores(models.Model):
+    """
+    Condition scores are computed from statistics aggregated across all relevant stands
+    within a project area or planning area.
+    """
+    # Either plan or project should be present.
+    plan = models.ForeignKey(Plan, on_delete=models.CASCADE, null=True)
+    project_area = models.ForeignKey(
+        ProjectArea, on_delete=models.CASCADE, null=True)
+    
+    # Condition
+    condition = models.ForeignKey(
```

Comment:
if you did this as a "ManyToMany" field as in line 53, would that eliminate the circular dependency?

Or maybe you're saying this shouldn't be done that way.

Reply:
I had a circular dependency because I didn't properly restore conditions/models.py.
It's ok for ConditionScores to live in plan/models.py.